Mathlib/Tactic/StacksAttribute.lean

```diff
@@ -143,12 +143,19 @@ syntax (name := stacksTag) stacksTagDB stacksTagParser (ppSpace str)? : attr
 initialize Lean.registerBuiltinAttribute {
   name := `stacksTag
   descr := "Apply a Stacks or Kerodon project tag to a theorem."
-  add := fun decl stx _attrKind => match stx with
-    | `(attr| stacks $tag $[$comment]?) => do
-      addTagEntry decl .stacks (← tag.getStacksTag) <| (comment.map (·.getString)).getD ""
-    | `(attr| kerodon $tag $[$comment]?) => do
-      addTagEntry decl .kerodon (← tag.getStacksTag) <| (comment.map (·.getString)).getD ""
-    | _ => throwUnsupportedSyntax
+  add := fun decl stx _attrKind => do
+    let oldDoc := (← findDocString? (← getEnv) decl).getD ""
+    let (database, url, tag, comment) := ← match stx with
+      | `(attr| stacks $tag $[$comment]?) =>
+        return ("Stacks", "https://stacks.math.columbia.edu/tag", tag, comment)
+      | `(attr| kerodon $tag $[$comment]?) =>
+        return ("Kerodon", "https://kerodon.net/tag", tag, comment)
+      | _ => throwUnsupportedSyntax
+    let tagStr ← tag.getStacksTag
+    let comment := (comment.map (·.getString)).getD ""
+    let newDoc := [oldDoc, s!"[{database} Tag {tagStr}]({url}/{tagStr})", comment]
+    addDocString decl <| "\n\n".intercalate (newDoc.filter (· != ""))
+    addTagEntry decl .stacks tagStr <| comment
```

Comment:
In the old code there can be `.kerodon`, but here you always use `.stacks`. Maybe you should return this at the `let (database, url, tag, comment) := ...` code.

Reply:
Ah, good point!  I think that this will also fix the tests.  I've been away from a computer since I pushed the latest commit, but I'll take care of this later.